browser/brave_profile_prefs.cc

```diff
@@ -54,7 +54,7 @@
 #endif
 
 #if BUILDFLAG(IPFS_ENABLED)
-#include "brave/browser/ipfs/ipfs_service.h"
+#include "brave/components/ipfs/browser/ipfs_service.h"
```

Comment:
I think you might be missing a dep here now

Reply:
I don't think it's missing a dep, there was one there already:

<img width="1099" alt="Screen Shot 2020-09-29 at 3 21 49 PM" src="https://user-images.githubusercontent.com/831718/94605747-8c14ea00-0267-11eb-8fcf-342ce4d1d0f1.png">